dagmc_stats/dagmc_stats.py

```diff
@@ -297,3 +297,157 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_tri_vert_data(my_core, native_ranges):
+    """
+    Build a numpy strcutured array to store triangle and vertex related data in the form of
+    triangle entity handle | vertex entity handle | angle connected to the vertex | side length of side opposite to vertex in the triangle
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    native_ranges : a dictionary containing ranges for each native type in the file (VERTEX, TRIANGLE, ENTITYSET)
+    
+    outputs
+    -------
+    tri_vert_data : a numpy structured array that stores the triangle and vertex related data
+    all_verts : (list) all the vertices that are connected to triangle in the geometry
+    """
+    
+    all_tris = native_ranges[types.MBTRI]
+    all_verts = []
+    
+    tri_vert_struct = np.dtype({'names':['tri','vert','angle','side_length'], 'formats':[np.uint64, np.uint64, np.float64, np.float64]})
+    tri_vert_data = np.zeros(0,dtype=tri_vert_struct)
+    
+    for tri in all_tris:
+        side_lengths = get_tri_side_length(my_core, tri)   # {vert : side_length}
+        side_length_sum_sq = sum(map(lambda i : i * i, side_lengths.values()))
+        side_length_prod = np.prod(side_lengths.values())
+        verts = list(my_core.get_adjacencies(tri, 0, op_type=1))
+        for vert_i in verts:
+            if vert_i not in all_verts:
+                all_verts.append(vert_i)
+            side_i = side_lengths[vert_i]
+            d_i = np.arccos((side_length_sum_sq - 2 * (side_i**2)) * side_i/(2 * side_length_prod))
+            bar = np.array((tri,vert_i,d_i,side_i),dtype=[('tri',np.uint64), ('vert',np.uint64), ('angle',np.float64), ('side_length',np.float64)])
+            tri_vert_data = np.append(tri_vert_data,bar)
```

Comment:
```suggestion
            tri_vert_data = np.append(tri_vert_data, bar)
```
PEP8 space change (there are other locations as well)